<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_number">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_bumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sex">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_sex">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_email">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_major">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>学历层次</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_email">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_passwod">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_emailcode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱验证码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_emailcode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_confirmPasswod">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_confirmPasswod">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>540</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_register">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>630</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>680</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_emailcode">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_department">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
